tutorials/host-github-runner/index.mdx

```diff
@@ -85,7 +85,7 @@ In this guide, you will learn how to configure a GitHub Actions runner on a Scal
     </Message>
 
     The configuration tool connects to GitHub using the given token.
-6. Enter the name of the runner during the runner configuration. It gets automatically several tags attached to it. Optionally you can enter custom tags. When prompted, enter the `work` directory of the runner. You can confirm the default values by pressing enter. Otherwise enter your customized values before pressing enter.
+6. Enter the name of the runner during the runner configuration. It gets automatically several tags attached to it. Optionally you can enter custom tags. When prompted, enter the `work` directory of the runner. You can confirm the default values by pressing enter. Otherwise, enter your customized values before pressing enter.
```

Comment:
```suggestion
6. Enter the name of the runner during the runner configuration. Several tags are automatically attached to it. Optionally, you can enter custom tags. When prompted, enter the `work` directory of the runner. You can confirm the default values by pressing enter. Otherwise, enter your customized values before pressing enter.
```